src/vs/platform/files/browser/indexedDBFileSystemProvider.ts

```diff
@@ -34,6 +34,7 @@ const ERR_FILE_NOT_FOUND = createFileSystemProviderError(localize('fileNotExists
 const ERR_FILE_IS_DIR = createFileSystemProviderError(localize('fileIsDirectory', "File is Directory"), FileSystemProviderErrorCode.FileIsADirectory);
 const ERR_FILE_NOT_DIR = createFileSystemProviderError(localize('fileNotDirectory', "File is not a directory"), FileSystemProviderErrorCode.FileNotADirectory);
 const ERR_DIR_NOT_EMPTY = createFileSystemProviderError(localize('dirIsNotEmpty', "Directory is not empty"), FileSystemProviderErrorCode.Unknown);
+const ERR_FILE_EXCEEDS_MEMORY_LIMIT = createFileSystemProviderError(localize('fileExceedsMemoryLimit', "File exceeds memory limit"), FileSystemProviderErrorCode.FileExceedsMemoryLimit);
```

Comment:
Should this be more specific to the storage quota? Memory limit to me sounds as if the RAM of the machine is not large enough to process the file.